src/framework/theme/components/select/option.component.ts

```diff
@@ -103,7 +106,7 @@ export class NbOptionComponent<T> implements OnDestroy {
 
   @HostBinding('attr.disabled')
   get disabledAttribute(): '' | null {
-    return this.disabled ? '' : null;
+    return (this.disabledByGroup || this.disabled) ? '' : null;
```

Comment:
Could you please split that line into multiple lines to make it more readable. Like so
```
const disabled = this.disabledByGroup || this.disabled;
return disabled ? '' : null;
```
Or even using if statement